3_javascript/javascript1/week2/javascript.js

```diff
@@ -0,0 +1,146 @@
+//Welcome to my second experience in JAVASCRIPT :)
+
+/* console.log("*******     Flight booking fullname function    *******");
+ function getFullname(firstname, surname) {
+     return `${firstname} ${surname}`;
+}
+
+ const fullname1 = getFullname("Helena", "Hashemi");
+ const fullname2 = getFullname("Shahram", "Shokri");
+
+ console.log(fullname1);
+ console.log(fullname2); */
+
+console.log("*******     Flight booking fullname function / ('Formal fullnam')     *******");
+function getFullname(firstname, surname, gender, userFormalName) {
+
+    if(gender === "male") {
```

Comment:
here you should check for `userFormalName` before you check the gender, think of it this way:
 if I want to use formal name => if the gender is `male` ,I will use `Lord`, else if the gender is `female` I will use `Lady`

Reply:
good explanation and thank you for reminding me to use "IF" "ELSE" :)
done!